services/dy-dash/cc-rabbit-0d/src/cc-rabbit-0d/cc-rabbit-0d.py

```diff
@@ -49,16 +38,68 @@ def download_all_inputs(n_inputs = 2):
 print('url_base_pathname', base_pathname)
 
 
-server = flask.Flask(__name__)
+server = Flask(__name__)
 app = dash.Dash(__name__,
     server=server,
     url_base_pathname=base_pathname
 )
+
+bp = Blueprint('myBlueprint', __name__)
+
+#---------------------------------------------------------#
+# Data to service
+data_paths = None
+
+@bp.route("/healthcheck")
+def healthcheck():
+    return Response("healthy", status=200, mimetype='application/json')
+
+def download_all_inputs(n_inputs = 2):
+    ports = node_ports.ports()
+    tasks = asyncio.gather(*[ports.inputs[n].get() for n in range(n_inputs)])
+    paths_to_inputs = asyncio.get_event_loop().run_until_complete( tasks )
+    return paths_to_inputs
+
+@bp.route("/retrieve", methods=['GET', 'POST'])
+def retrieve():
+    global data_paths
```

Comment:
normally POST should check for the port keys and only retrieve those. but in the casse of this dash there is only one input right?